src/Facet.php

```diff
@@ -141,6 +141,11 @@ private function fetchItems($name = NULL, $dateGranularity = 'year') {
             $name = $this->getName();
         }
 
+        //  Facet items are shown with defined minimum count for each facet.
+        $miniCountColumns = $this->getApp()->settings('minimum facet counts');
+        $miniCount = $miniCountColumns[$name];
```

Comment:
@jhuang0427 If $name is not in $miniCountColumns, I think this will give a PHP warning/error. Can we check first that $name is in $miniCountColumns?

Reply:
I had checked w/o in array, for instance usdojaccounttype, the return is null as detected to be 0

Reply:
@jhuang0427 But the line:
```
$miniCount = $miniCountColumns[$name]
```

If $name is not in $miniCount, that will produce a "PHP Notice: Undefined index"

Reply:
yes, an exception fired, I am going to correct the codes